pkgs/development/compilers/bigloo/default.nix

```diff
@@ -13,9 +13,10 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ autoconf automake libtool ];
 
-  buildInputs = lib.optional stdenv.isDarwin
+  buildInputs = lib.optionals stdenv.isDarwin [
     darwin.apple_sdk.frameworks.ApplicationServices
```

Comment:
darwin should not be used as an input

Reply:
i agree but not really in the scope of this pr